config/kbin_routes/front.yaml

```diff
@@ -1,85 +1,94 @@
 front:
   controller: App\Controller\Entry\EntryFrontController::front
-  defaults: { subscription: home, sortBy: hot, time: '∞', type: all, federation: all, content: threads }
-  path: /{subscription}/{sortBy}/{time}/{type}/{federation}/{content}
+  defaults: &front_defaults { subscription: home, content: threads, sortBy: hot, time: '∞', federation: all }
+  path: /{subscription}/{content}/{sortBy}/{time}/{federation}
   methods: [GET]
-  requirements:
+  requirements: &front_requirement
     subscription: "%default_subscription_options%"
     sortBy: "%default_sort_options%"
     time: "%default_time_options%"
-    type: "%default_type_options%"
     federation: "%default_federation_options%"
     content: "%default_content_options%"
 
-front_redirect:
-  controller: App\Controller\Entry\EntryFrontController::front_redirect
-  defaults: { sortBy: hot, time: '∞', type: all, federation: all, content: threads }
-  path: /{sortBy}/{time}/{type}/{federation}/{content}
+front_sub:
+  controller: App\Controller\Entry\EntryFrontController::front
+  defaults: *front_defaults
+  path: /{subscription}/{sortBy}/{time}/{federation}
   methods: [GET]
-  requirements:
-    sortBy: "%default_sort_options%"
-    time: "%default_time_options%"
-    type: "%default_type_options%"
-    federation: "%default_federation_options%"
-    content: "%default_content_options%"
+  requirements: *front_requirement
```

Comment:
I appreciate how you find these things that can be refactored into less copy paste

Reply:
thankfully this is one of the simpler ones, yaml anchors can be quite confusing at time, especially when you also add merge keys to the mix

technically this could go a step further by using merge keys referencing first full front route definition then overwrite `path` with different routes (iirc the old front routing before ui filter patch actually does this)